gst/nnstreamer/tensor_if/gsttensorif.c

```diff
@@ -166,6 +166,7 @@ gst_tensor_if_cv_get_type (void)
       {TIFCV_A_VALUE, "A_VALUE", "Decide based on a single scalar value"},
       {TIFCV_TENSOR_AVERAGE_VALUE, "TENSOR_AVERAGE_VALUE",
           "Decide based on a average value of a specific tensor"},
+      {TIFCV_CUSTOM, "CUSTOM", "Decide based on a user defined condtion"},
```

Comment:
I think it is good to flesh out the word condition.
```suggestion
      {TIFCV_CUSTOM, "CUSTOM", "Decide based on a user defined callback"},
```

Reply:
Updated.